docs/build_process.md

```diff
@@ -49,7 +49,7 @@ registry_uri = registry.example.com
  * for backwards compatibility reasons, it is possible to suffix `registry_uri` with `/v2`.
 
 ```ini
-registry_uri = registry.example.com/v2
+registry_uri = registry.example.com
```

Comment:
either leave '/v2' here OR
delete all that, because "registry_uri = registry.example.com" is already at line 41:
 * for backwards compatibility reasons, it is possible to suffix `registry_uri` with `/v2`.

```ini
registry_uri = registry.example.com

Reply:
Thanks, fixed.